RetailCoder.VBE/UI/CodeInspections/CodeInspectionsDockablePresenter.cs

```diff
@@ -176,27 +176,19 @@ private void Control_RefreshCodeInspections(object sender, EventArgs e)
             Control.EnableRefresh(false);
             Control.Cursor = Cursors.WaitCursor;
 
-            try
```

Comment:
I'm wondering why we removed the try here. Been there done that and there's a cursor bug just waiting to happen here. The cursor *needs* to be set back to default in the finally block IIRC.